Sources/Sentry/include/SentryTracer.h

```diff
@@ -98,9 +98,16 @@ NS_ASSUME_NONNULL_BEGIN
  * Finishes the span by setting the end time and span status.
  *
  * @param status The status of this span
- *  */
+ */
 - (void)finishWithStatus:(SentrySpanStatus)status NS_SWIFT_NAME(finish(status:));
 
+/**
+ * Returns the latest span that is not finished.
+ *
+ * @return span or null if none.
+ */
+- (nullable id<SentrySpan>)getLatestActiveSpan;
```

Comment:
I know this exist in Java but I'm not sure about C#, Go, Python.

Is this part of the spec?
I assume it is but if not, before we add this API here too can we discuss this in details?

Reply:
C# does

https://github.com/getsentry/sentry-dotnet/blob/7186bec5d11254afb967ee1d1f91724270294e13/src/Sentry/Scope.cs#L388-L392

looks like Python has a dedicated field for the Active Span within the Scope

https://docs.sentry.io/platforms/python/performance/custom-instrumentation/#retrieve-the-current-span

Reply:
Good chance to keep this out then if it's not in the spec, specially if we don't use it.
Then we just need to deal with .NET and Java later.